### DISCUSSION TITLE
Curly quotes

### DISCUSSION THREAD

**Discussion by mustafa0x** [micromark/micromark#53] [Q&A] [answered]
I've searched and didn't find a plugin that converts straight quotes to curly quotes [1] .  Does such a plugin exist?

[1]: Not part of of the CommonMark spec, but a common feature in Markdown tools, e.g. pandoc and markdown-it.

**Comment by wooorm** [comment, marked as answer]
Use the higher-level tools: mdast and remark. E.g., https://github.com/remarkjs/remark-textr, or https://github.com/silvenon/remark-smartypants. Or do it yourself.

See https://github.com/micromark/micromark#comparison for more information.
